src/components/onboarding/Wizard.vue

```diff
@@ -134,10 +134,8 @@ export default {
       // set step after the last to hide permanently (or until a new 'step' is added eventually):
       this.setWizardIsCollapsed(true);
 
-      this.setWizardCurrentStep(steps.length);
-
-
-      if (this.account) {
+      if (this.account && this.account.length) {
+        this.setWizardCurrentStep(steps.length);
```

Comment:
Why is that? this will break the wizard from going forward. If it is intentional, changing steps from the top navigation will still bypass this logic